kie-server-parent/kie-server-common/src/main/java/org/kie/server/common/KeyStoreHelperUtil.java

```diff
@@ -7,14 +7,44 @@
 import org.slf4j.LoggerFactory;
 
 public class KeyStoreHelperUtil {
+    // the private key identifier for REST controller
+    private static final String PROP_PWD__REST_ALIAS = "kie.keystore.key.rest.alias";
+    // the private key identifier for REST controller
+    private static final String PROP_PWD__REST_PWD = "kie.keystore.key.rest.pwd";
```

Comment:
same applies here for properties - there is either server (kie server) password or controller password.